src/client/java/teammates/client/scripts/DataMigrationForComments.java

```diff
@@ -9,9 +9,9 @@
 import javax.jdo.Query;
 
 import teammates.client.remoteapi.RemoteApiClient;
-import teammates.common.datatransfer.CommentParticipantType;
-import teammates.common.datatransfer.CommentStatus;
-import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.datatransfer.misc.CommentParticipantType;
```

Comment:
Hmm that's alright too, i put the *bundle files into .attributes because they seem to be importing mostly attribute files